src/RemoteSymbolProvider/MicrosoftSymbolServerSymbolProviderTest.cpp

```diff
@@ -93,11 +95,13 @@ TEST_F(MicrosoftSymbolServerSymbolProviderTest, RetrieveModuleSuccess) {
 
   orbit_base::StopSource stop_source{};
   symbol_provider_.RetrieveSymbols(kValidModuleId, stop_source.GetStopToken())
-      .Then(executor_.get(), [this](ErrorMessageOr<CanceledOr<std::filesystem::path>> result) {
-        EXPECT_THAT(result, HasNoError());
-        EXPECT_FALSE(IsCanceled(result.value()));
-        const auto& local_file_path = orbit_base::GetNotCanceled(result.value());
-        EXPECT_EQ(local_file_path, symbol_cache_.GenerateCachedFilePath(kValidModuleId.file_path));
+      .Then(executor_.get(), [this](const SymbolLoadingOutcome& result) {
+        EXPECT_TRUE(orbit_symbol_provider::IsSuccessResult(result));
```

Comment:
```suggestion
        ASSERT_TRUE(orbit_symbol_provider::IsSuccessResult(result));
```
So it fails. Otherwise the next call to GetSuccessResult will crash